lib/__tests__/standalone-fix.test.mjs

```diff
@@ -39,35 +37,16 @@ it('fixes when enabled in config', async () => {
 	expect(output).toBe('a { color: #fff; }');
 });
 
-it('apply indentation autofix at last', async () => {
-	const result = await standalone({
-		code: 'a {\nbox-shadow: 0 -1px 0 0 rgba(0, 0, 0, 0.1), 0 0 0 1px rgba(0, 0, 0, 0.2), inset 0 1px 2px 0 rgba(0, 0, 0, 0.1);\n}',
-		config: {
-			rules: {
-				indentation: 2,
-				'value-list-comma-newline-after': 'always',
-			},
-		},
-		fix: true,
-	});
-
-	expect(result.output).toBe(
-		'a {\n  box-shadow: 0 -1px 0 0 rgba(0, 0, 0, 0.1),\n    0 0 0 1px rgba(0, 0, 0, 0.2),\n    inset 0 1px 2px 0 rgba(0, 0, 0, 0.1);\n}',
-	);
-});
-
```

Comment:
This seems like an `indentation`-specific test, so I deleted it. I can re-add a test if there's a more general behaviour we're testing!

Reply:
It appears that this test shouldn't depend on specific rules. 🤔 
Is it testing how multiple rules' autofix works?

Reply:
> Is it testing how multiple rules' autofix works?

I believe it's testing the workaround we added so that the `indentation` rule runs last. (It was one of the shortcomings of the standalone stylistic rule approach we took.)

I think we're good to delete the test.